docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,1035 @@
+            subgraph HCP[Hosted Control Plane]
+                DescHCP("Contains the OCP
+                HostedCluster details")
+            end
+            subgraph AWSCL[AWSCluster]
+                DescAWSC("Contains the AWS details like
+                VPC, Region, Subnet, etc...")
+            end
+            subgraph CL[Cluster]
+                DescCL("Perform the relationship between
+                AWSCluster and HostedControlPlane")
+            end
+            subgraph AWSMT[AWS Machine Template]
+                DescAMT("It's a template used to
+                create the AWSMachines")
+            end
+            subgraph AWSM[AWS Machines]
+                DescAM("These are the representation of
+                AWS EC2 instances at Openshift level")
+            end
+            subgraph MCH[Machines]
+                DescMCH("Final machine definition
+                which links with AWS Machine")
+            end
+            subgraph MDPL[MachineDeployment]
+                DescMDPL("Final machine definition
+                which links with AWS Machine")
+            end
+            subgraph MCHS[MachineSet]
+                DescMCHS("Final machine definition
+                which links with AWS Machine")
+            end
+        end
+    end
+```
+!!! note "Legend"
+    - **Orange:** OCP objects directly related with AWS.
+    - **Cyan:** Provider agnostic OCP objects.
+    - The *dotted* lines are associations among AWS Components created by the CLI or the Operator
+    - The **thick** lines are objects created by the CLI into AWS platform and needed by the Hypershift Operator to work properly
+
+Also the NodePool Controller calling tha AWS Cluster API provider, will create some objects in AWS which are these ones:
+
+- **Diagram:** The diagram shows the NodePool resultant objects created into the AWS Infrastructure.
+
+```mermaid
+flowchart LR
+    style HO fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style EC2 fill:#FEA175,stroke:#333,stroke-width:3px
+    style IAM fill:#FEA175,stroke:#333,stroke-width:3px
+    subgraph OCP["Openshift Management Cluster"]
+        direction LR
+        HO[fa:fa-server Hypershift Operator]
+    end
+    HO ==> EC2
+    HO ==> IAM
+    EC2I1 -.-> IAMR
+    EC2I2 -.-> IAMR
+    EC2I1 -.-> IAMP
+    EC2I2 -.-> IAMP
+    subgraph ACP["AWS Cloud Provider"]
+        direction LR
+        subgraph EC2
+            EC2I1[EC2 Instance]
+            EC2I2[EC2 Instance]
+        end
+        subgraph IAM
+            IAMR[Role Policy]
+            IAMP[Profile]
+        end
+    end
+
+```
+
+!!! note "Legend"
+    - **Orange:** OCP objects directly related with AWS.
+    - **Cyan:** Provider agnostic OCP objects.
+    - The *dotted* lines are associations among AWS Components created by the CLI or the Operator
+    - The **thick** lines are objects created by the CLI into AWS platform and needed by the Hypershift Operator to work properly
+
+## AWS Components and Hypershift
```

Comment:
This is the part that I'm interested in. Can we please put this in its own section within reference/provider-infrastructure